runtime/cudaq/platform/default/rest/helpers/oqc/OQCServerHelp.cpp

```diff
@@ -118,16 +118,17 @@ OQCServerHelper::get_from_config(BackendConfig config, const std::string &key,
   return iter != backendConfig.end() ? iter->second : default_return;
 }
 
-// Retrieve an environment variable
+/// @brief Retrieve an environment variable. Return empty string if variable
+/// does not exist.
 std::string OQCServerHelper::getEnvVar(const std::string &key) const {
   // Get the environment variable
   const char *env_var = std::getenv(key.c_str());
-  // If the variable is not set, throw an exception
-  if (env_var == nullptr) {
-    throw std::runtime_error(key + " environment variable is not set.");
-  }
-  // Return the variable as a string
-  return std::string(env_var);
+  // Return the variable as a string. Initializing with a null pointer is
+  // undefined, so use empty constructor if necessary.
```

Comment:
Does an empty string behave correctly? Do we need an error in this case?

Reply:
> Does an empty string behave correctly?

If the environment variable is an empty string, the `std::string(env_var)` constructor will behave as expected. That is, an empty string will be constructed.

> Do we need an error in this case?

The error notifying the user that an empty string is not a valid username is already handled by line 331. (I just did a test to confirm, too.)